<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calibration_crystal_editor</class>
 <widget class="QWidget" name="calibration_crystal_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Overlay</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDoubleSpinBox:disabled {background-color: LightGray;}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="calibration_crystal_group_box">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Calibration Crystal</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QTabWidget" name="tab_widget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="form_tab">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>Form</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="2">
           <widget class="ScientificDoubleSpinBox" name="position_1">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>8</number>
            </property>
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="4">
           <widget class="QGroupBox" name="stretch_matrix_group_box">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Stretch Matrix</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="1">
              <widget class="ScientificDoubleSpinBox" name="stretch_matrix_7">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="ScientificDoubleSpinBox" name="stretch_matrix_4">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="ScientificDoubleSpinBox" name="stretch_matrix_3">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="ScientificDoubleSpinBox" name="stretch_matrix_2">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="ScientificDoubleSpinBox" name="stretch_matrix_0">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="suffix">
                <string/>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="ScientificDoubleSpinBox" name="stretch_matrix_1">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="ScientificDoubleSpinBox" name="stretch_matrix_5">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="ScientificDoubleSpinBox" name="stretch_matrix_8">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="ScientificDoubleSpinBox" name="stretch_matrix_6">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="ScientificDoubleSpinBox" name="position_2">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>8</number>
            </property>
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="ScientificDoubleSpinBox" name="orientation_1">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="decimals">
             <number>8</number>
            </property>
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="orientation_label">
            <property name="text">
             <string>Orientation:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="position_label">
            <property name="text">
             <string>Position:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ScientificDoubleSpinBox" name="orientation_0">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="decimals">
             <number>8</number>
            </property>
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ScientificDoubleSpinBox" name="position_0">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>8</number>
            </property>
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="ScientificDoubleSpinBox" name="orientation_2">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="decimals">
             <number>8</number>
            </property>
            <property name="minimum">
             <double>-1000000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="slider_tab">
         <attribute name="title">
          <string>Slider</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QWidget" name="slider_widget_parent" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="refinements_tab">
         <attribute name="title">
          <string>Refinements</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="refinements_selector_layout"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="load">
        <property name="text">
         <string>Load Crystal Parameters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tab_widget</tabstop>
  <tabstop>orientation_0</tabstop>
  <tabstop>orientation_1</tabstop>
  <tabstop>orientation_2</tabstop>
  <tabstop>position_0</tabstop>
  <tabstop>position_1</tabstop>
  <tabstop>position_2</tabstop>
  <tabstop>stretch_matrix_0</tabstop>
  <tabstop>stretch_matrix_1</tabstop>
  <tabstop>stretch_matrix_2</tabstop>
  <tabstop>stretch_matrix_3</tabstop>
  <tabstop>stretch_matrix_4</tabstop>
  <tabstop>stretch_matrix_5</tabstop>
  <tabstop>stretch_matrix_6</tabstop>
  <tabstop>stretch_matrix_7</tabstop>
  <tabstop>stretch_matrix_8</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
